microscopy_SPIM001/sub-01/microscopy/sub-01_sample-A_stain-LFB_chunk-02_SPIM.json

```diff
@@ -0,0 +1,16 @@
+{
+        "Manufacturer": "Miltenyi Biotec",
+        "ManufacturersModelName": "UltraMicroscope II",
+        "BodyPart": "CSPINE",
+        "SampleEnvironment": "exvivo",
```

Comment:
Another update to do in both datasets is to change `"exvivo"` for `"ex vivo"` in all json files.
Just an observation, feel free to do it here or in another PR as you prefer.

Reply:
Done